packages/cli/src/utils/command.ts

```diff
@@ -4,14 +4,24 @@ import { IArg, ParseFn } from '@oclif/parser/lib/args'
 import { pathExistsSync } from 'fs-extra'
 import Web3 from 'web3'
 
-const parsePublicKey: ParseFn<string> = (input) => {
-  // Check that the string starts with 0x and has byte length of ecdsa pub key (64 bytes) + bls pub key (48 bytes) + proof of pos (96 bytes)
-  if (Web3.utils.isHex(input) && input.length === 418 && input.startsWith('0x')) {
+const parseBytes = (input, length, msg) => {
+  // Check that the string starts with 0x and has byte length of `length`.
+  if (Web3.utils.isHex(input) && input.length === length && input.startsWith('0x')) {
     return input
   } else {
-    throw new CLIError(`${input} is not a public key`)
+    throw new CLIError(msg)
   }
 }
+
+const parseEcdsaPublicKey: ParseFn<string> = (input) => {
+  parseBytes(input, 64, `${input} is not an ECDSA public key`)
+}
+const parseBlsPublicKey: ParseFn<string> = (input) => {
+  parseBytes(input, 48, `${input} is not a BLS public key`)
```

Comment:
Can we pull these numbers from a common place? Especially since they might change soon.

Reply:
Added constants in `@celo/utils/src/bls`